contribute-docs/style-guide/formatting.md

```diff
@@ -0,0 +1,485 @@
+
+## Emphasis
+
+Bold, italic, and monospace formatting helps users distinguish words and phrases from the surrounding text and provides visual cues for users. 
+
+Use emphasis only in the text body, and avoid using it in titles and links. All titles must appear uniform so that users can scan them in the table of contents. By default, links are emphasized by rendering in a different color. Avoid applying multiple variations of emphasis to the same text. For example, when link text includes the name of a UI element, avoid using bold.
+
+### Bold / strong
+
+Use bold text formatting to emphasize the names of UI elements so that users can identify interactive components. 
+
+| Element | Example |
+| ------- | ------- |
+| Apps | **Visualize** allows you to create visualizations of the data in your Elasticsearch indices. |
+| Columns | In the **Value** column, go to the value you want to edit and click **Edit**. |
+| Interactive UI functions | To use a dark color theme, click **Options** and select **Use dark theme**. |
+| Key combinations | Press **Alt+C**. |
+| Menu items | In Kibana, open the main menu, then click **Discover**. |
+| Page names | The **Hosts** page provides a comprehensive overview of all hosts and host-related security events. |
+| Sections | The **Document Summary** section provides general alert details, including any available threat intelligence data. |
+| Tables | In the **Value lists** table, click the value list you want to edit. |
+| Tabs | The **Overview** tab provides an overview of the alert and shows relevant threat intelligence details. |
+
+:::::{note}
+If the name of the UI element isn't explicitly shown in the UI, then don't bold it.
+::::{dropdown} Example
+The risk summary table shows the category, score, and number of risk inputs that determine the host risk score.
+::::
+:::::
+
+### Italic / emphasis
+
+Use italic text formatting to emphasize new words and concepts that are introduced to users for the first time, as well as titles of Elastic documentation resources.
+
+| Element | Example |
+| ------- | ------- |
+| Terms | A Metricbeat *module* defines the basic logic for collecting data from a specific service. |
+
+### Monospace / code
+
+Use monospace text formatting to emphasize technical elements to help users differentiate code and command examples from regular text.
+
+| API endpoints | The endpoints include `/_bulk` and `/INDEX/_bulk`. |
+| --- | --- |
+| Class names | Date fields are exposed as `ReadableDateTime`, so they support methods like `getYear`. |
+| Code blocks | ```sh       GET _tasks       GET _tasks?nodes=nodeId1,nodeId2       GET _tasks?nodes=nodeId1,nodeId2&actions=cluster:*       ``` |
+| Command invocations | To create the basic files for your metricset, run `make create-metricset`. |
+| Command names | After you start Logstash, wait until you see `Pipeline main started`. |
```

Comment:
That example seems like a message instead of a command. Here's an alternative:
```suggestion
| Command names |  The `elasticsearch-certutil` command simplifies the creation of certificates. |
```